### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by b-rage**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <style>
         body {
             background-color: orange;
-            border: 10px solid skyblue;
+            border: 10px solid greenyellow;
```
Comment (calvinoxsn) [inline index.html]: gianluca, why "greenyellow" and not an RGB value? =)

Comment (b-rage) [reply index.html]: it's fast :D

File: index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <style>
```
Comment (calvinoxsn) [inline index.html]: Hey, we should move the styles to a different file!

Comment (b-rage) [reply index.html]: true, but both should have do taht, how we procede then?

Comment (calvinoxsn) [reply index.html]: Ok, you're right, let's do whatever's necessary!

Comment (calvinoxsn) [reply index.html]: Don't worry, I'l move the syles to a separate .css file!

**Pull request merged**